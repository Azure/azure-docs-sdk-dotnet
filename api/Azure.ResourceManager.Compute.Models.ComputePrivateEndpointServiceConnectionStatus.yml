### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus&text=ComputePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus,Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.Compute.Models
name: ComputePrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.Compute.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Compute v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Compute v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct ComputePrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type ComputePrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure ComputePrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of ComputePrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.Compute.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::Compute::Models::ComputePrivateEndpointServiceConnectionStatus
  - ComputePrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
