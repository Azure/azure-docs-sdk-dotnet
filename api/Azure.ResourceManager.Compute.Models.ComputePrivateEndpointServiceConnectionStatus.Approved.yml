### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus&text=ComputePrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  nameWithType: ComputePrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As ComputePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.Compute.Models
name: Approved
fullName: Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
nameWithType: ComputePrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.Compute.dll
packagesWithMoniker:
- value: Azure.ResourceManager.Compute v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.Compute v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.Compute.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.Compute.Models.ComputePrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::Compute::Models::ComputePrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - ComputePrivateEndpointServiceConnectionStatus.Approved
  - ComputePrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - ComputePrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct ComputePrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
